ukaea-schema/facility/hive/port.schema.json

```diff
@@ -19,7 +19,7 @@
         },
         "portSizeStandard": {
             "type": "string",
-            "example": "CF100"
+            "example": "CF100" 
```

Comment:
I am guessing this has been run through a linter/JSON validator, we probably want to remove this whitespace

Reply:
This suggests to me that #10 should be next on the agenda.